lib/jekyll/readers/post_reader.rb

```diff
@@ -76,7 +76,7 @@ def processable?(doc)
     def publishable?(doc)
       site.publisher.publish?(doc).tap do |will_publish|
         if !will_publish && site.publisher.hidden_in_the_future?(doc)
-          Jekyll.logger.warn "Skipping:", "#{doc.relative_path} has a future date"
+          Jekyll.logger.warn "Skipping:", "#{doc.relative_path} has a future date, run jekyll with --future to publish anyway"
```

Comment:
```suggestion
          Jekyll.logger.warn "Skipping:", "#{doc.relative_path} has a future date. You can pass  --future option if you want to publish posts with a future date."
```

Reply:
I like the idea of providing docs from the CLI, but here this might be too verbose and not a so common use case.

@jekyll/core Do we really need to enhance our warning here?